pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Description
+
+Include a summary of the change and what was done. Also include relevant motivation and context for the change. If any, list dependencies are required for this change.
+
+Include the Asana link, or relevant issue. (https://app.asana.com/0/187002107997407/list)
+
+Don't forget to add reviewers on github and mention the PR in the appropriate discord channel.
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions and details so the changes can be tested if required. Also mention who performed the testing.
+
+# Documentation
+
+Include links to changes in documentation on confluence that you have created, if it was required.
+
+https://uwarg-docs.atlassian.net/wiki/spaces/ZP/overview?homepageId=20217925
+
+# Checklist:
+
+- [ ] The changes have been well commented, particularly in hard-to-understand areas.
+- [ ] The code has been tested on hardware, either by me or somone else.
+- [ ] Corresponding changes to documentation have been created links to this documentation are provided within the pull request.
+- [ ] The changes generate no new warnings and compile and run; A screenshot of a successful compile message is attached to the PR.
```

Comment:
So is this checklist to create the PR or to merge? For merging its perfect, however I would think that a PR can be created without having all of these checked off as well, since it can be a draft PR or a work in progress

Reply:
I was thinking it was a checklist for merge. I think you can edit checkboxes after a PR is opened, so that shouldn't be an issue. I'll clarify the language in the document.